packages/vitepress-plugin-git-changelog/src/vite/helpers.test.ts

```diff
@@ -84,130 +84,106 @@ describe('parseGitLogRefsAsTags', () => {
-      author_name: 'First Last',
-      author_email: 'user@example.com',
-    }
-
-    const commit = await initCommitWithFieldsTransformed(
-      mockedCommit,
+    const mockedCommits = ['c16db1033fce57f50b261e9944c136a26fcaccc6|Neko Ayaka|neko@ayaka.moe|Mon Mar 25 20:05:53 2024 +0800|release: v1.24.3| (tag: v1.24.3)|Signed-off-by: Neko Ayaka <neko@ayaka.moe>\n']
+
+    const commit = await parseCommits(
+      '',
+      mockedCommits,
       () => 'https://github.com/example-org/example',
       defaultCommitURLHandler,
       defaultReleaseTagURLHandler,
       defaultReleaseTagsURLHandler,
     )
 
-    expect(commit).toEqual({
-      paths: [],
-      tag: 'v1.0.0',
-      tags: ['v1.0.0'],
-      release_tag_url: 'https://github.com/example-org/example/releases/tag/v1.0.0',
-      release_tags_url: ['https://github.com/example-org/example/releases/tag/v1.0.0'],
-      hash: '0e4565940b08cc145d378169e85c37eca20d6036',
-      hash_url: 'https://github.com/example-org/example/commit/0e4565940b08cc145d378169e85c37eca20d6036',
-      date: '2023-11-10T11:59:06+08:00',
-      date_timestamp: 1699588746000,
-      message: 'release: v1.0.0',
-      refs: 'tag: v1.0.0',
-      body: 'Signed-off-by: First Last <user@example.com>\n',
-      author_name: 'First Last',
-      author_email: 'user@example.com',
-      author_avatar: 'b4c9a289323b21a01c3e940f150eb9b8c542587f1abfd8f0e1cc1ffc5e475514',
+    expect(commit).toEqual([{
+      path: '',
+      tag: 'v1.24.3',
+      tags: ['v1.24.3'],
+      release_tag_url: 'https://github.com/example-org/example/releases/tag/v1.24.3',
+      release_tags_url: ['https://github.com/example-org/example/releases/tag/v1.24.3'],
+      hash: 'c16db1033fce57f50b261e9944c136a26fcaccc6',
+      hash_url: 'https://github.com/example-org/example/commit/c16db1033fce57f50b261e9944c136a26fcaccc6',
+      date: 'Mon Mar 25 20:05:53 2024 +0800',
+      date_timestamp: 1711368353000,
+      message: 'release: v1.24.3',
+      body: 'Signed-off-by: Neko Ayaka <neko@ayaka.moe>\n',
+      author_name: 'Neko Ayaka',
+      author_email: 'neko@ayaka.moe',
+      author_avatar: '8002ecc992a1c4aba4c49fd04ea4b48ecf2468b85d3b9cdd6dae4163bf1bb945',
       repo_url: 'https://github.com/example-org/example',
-    })
+    }])
   })
 
   it('should transform for commit contains no refs', async () => {
-    const mockedCommit = {
-      hash: '77be05bfb36ce894638648bf5530122b31b951f8',
-      date: '2023-10-29T19:52:13+08:00',
-      message: 'docs: updated README.md',
-      refs: '',
-      body: 'Signed-off-by: First Last <user@example.com>\n',
-      author_name: 'First Last',
-      author_email: 'user@example.com',
-    }
-    const commit = await initCommitWithFieldsTransformed(
+    const mockedCommit = ['62ef7ed8f54ea1faeacf6f6c574df491814ec1b1|Neko Ayaka|neko@ayaka.moe|Wed Apr 24 14:24:44 2024 +0800|docs: fix english integrations list||Signed-off-by: Neko Ayaka <neko@ayaka.moe>\n']
```

Comment:
```suggestion
    const mockedCommit = ['62ef7ed8f54ea1faeacf6f6c574df491814ec1b1|First Last |user@example.com|Wed Apr 24 14:24:44 2024 +0800|docs: fix english integrations list||Signed-off-by: First Last <user@example.com>\n']
```

单测的时候我们还是中立一点，就不包含我自己的个人信息了，这样如果别人需要照抄的话也可以轻松复制过去。